dashboard/src/app/workspaces/workspace-details/select-stack/recipe-authoring/workspace-recipe-authoring.html

```diff
@@ -14,6 +14,7 @@
                  ng-required>
         <che-error-messages che-message-scope="workspace-details-environment"
                             che-message-name="Recipe content">
+          <div ng-message></div>
```

Comment:
This is a quick fix for the page crashed error. I Found that the first 'ng-message' error crashing problem during 'md-tab' select ( in case with 'ng-if' inside
https://github.com/eclipse/che/blob/master/dashboard/src/app/workspaces/workspace-details/select-stack/workspace-select-stack.html#L41).

We need to investigate it more. I added a comment.